### PR TITLE
Run tests against multiple versions of Ember

### PR DESCRIPTION
**Pull request by GavinJoyce**
/cc @rwjblue @stefanpenner 

This adds support for running tests against multiple version of Ember and closes https://github.com/eviltrout/ember-performance/issues/59.

The `TestSession` now contains an array of `TestGroup`s which have an associated Ember version and an array of `TestItem`s.

image

Test results are now presented as ascii text only (closes https://github.com/eviltrout/ember-performance/issues/62):

image

---

I'll create some follow-up issues once this has merged:

 * update app to use ember-cli (https://github.com/eviltrout/ember-performance/pull/64)
 * better results UI (group tests so that they can be compared easily)
 * support multi submit results (a single result can still be submitted)
 * Add Ember 2.1.0 (https://github.com/eviltrout/ember-performance/pull/63)
 * create issue for large error % in complex list rendering test
 * truncate long test names to avoid dodgy result formatting
 * adds graphs

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -48,28 +44,30 @@ <h3 class="text-muted">Ember Performance</h3>
...
-                <th>Name</th>
-                <th class="numeric">Speed</th>
-                <th class="numeric">Error</th>
-                <th class="numeric">Samples</th>
-                <th class="numeric">Mean</th>
-              </tr>
-            </thead>
-            <tbody>
-              {{#each result in report.results}}
+          {{#each testGroupReport in report.testGroupReports}}
```
Comment (rwjblue) [inline app/index.html]: Any reason not to use block params here?

File: app/index.html [hunk 2]
```diff
@@ -48,28 +44,30 @@ <h3 class="text-muted">Ember Performance</h3>
...
+                  <th class="numeric">Error</th>
+                  <th class="numeric">Samples</th>
+                  <th class="numeric">Mean</th>
                 </tr>
-              {{/each}}
-            </tbody>
-          </table>
+              </thead>
+              <tbody>
+                {{#each result in testGroupReport.results}}
```
Comment (rwjblue) [inline app/index.html]: Can we use block params here?

File: app/index.html (hunk 1 above)
Comment (GavinJoyce) [inline app/index.html]: We're on an older version of Ember, I'll update it after this PR lands

**Comment by rwjblue** [comment]
This is looking really good, thanks for putting the time into it.

RE: the checklist above, let us know when you think this is ready to land and I will review again.  We can make new issues for any remaining items (you mentioned some might be separate PR's).

File: app/index.html (hunk 1 above)
Comment (rwjblue) [inline app/index.html]: Gotcha.  Thank you for clarifying!

**Comment by GavinJoyce** [comment]
image

Challenge accepted

**Comment by stefanpenner** [comment]
wohoo!

**Comment by GavinJoyce** [comment]
@rwjblue this is ready for review, thanks

**Comment by eviltrout** [comment]
Looks fantastic to me too! Thanks so much for this feature.

**Comment by GavinJoyce** [comment]
I've got a couple of PRs backing up, it would be great if someone could review this

**Comment by rwjblue** [comment]
Sorry for the delay, got a bit behind the last couple days.

**Comment by GavinJoyce** [comment]
No worries, thanks!

**Pull request merged**